openpype/settings/entities/schemas/projects_schema/schemas/schema_maya_create.json

```diff
@@ -63,6 +63,11 @@
                     "type": "text",
                     "key": "default_render_image_folder",
                     "label": "Default render image folder"
+                },
+                {
+                    "type": "boolean",
+                    "key": "disable_all_lights",
+                    "label": "Disable Rendering of All Lights"
```

Comment:
I feel like this needs a clarification that this is the case for "Render Setup Layers"

With [this comment taken into consideration](https://github.com/pypeclub/OpenPype/pull/3038/files#r861015724) I feel like this could be labeled or documented: "Include all lights in Render Setup Layers by default" so it's very close to what Maya shows in the [menu entry as visible in screenshot in this comment](https://github.com/pypeclub/OpenPype/pull/3038#pullrequestreview-954451610).

Reply:
Done!